Makefile

```diff
@@ -165,7 +169,13 @@ vm: $(VM_IMAGE)
 # sure VM_IMAGE is as close as possible to the host!
 vm-local-repos: vm
 	bots/image-customize -v \
-		--upload /etc/yum.repos.d:/etc/yum.repos.d/ \
+		--run-command "rm -rf /etc/yum.repos.d" \
+		$(TEST_OS)
+	bots/image-customize -v \
+		--upload $(REPOS_DIR):/etc/yum.repos.d \
+		--run-command "yum -y remove composer-cli lorax-composer" \
+		--run-command "yum -y install composer-cli lorax-composer" \
+		--run-command "systemctl enable lorax-composer" \
 		--run-command "yum -y update" \
```

Comment:
I'd recommend to first remove, then update, then install. Otherwise the update may change the version again to something unintended?